src/dda/env/dev/types/linux_container.py

```diff
@@ -98,6 +109,9 @@ def start(self) -> None:
                 "-e",
                 AppEnvVars.TELEMETRY_API_KEY,
             ]
+            if self.config.arch is not None:
+                command.extend(["--platform", f"linux/{self.config.arch}"])
```

Comment:
Inconsequential performance nit:

```suggestion
                command.extend(("--platform", f"linux/{self.config.arch}"))
```